### DISCUSSION TITLE
View used to have "Friendly name" ...

### DISCUSSION THREAD

**Discussion by Eddie-BS** [jziolkowski/tdm#134] [Q&A] [answered]
Hi,
I am running 0.2.9 on OSX 10.15.6 and it runs nice, great work.
In previous versions I had a column displaying "Friendly Name" and I would like to see that one again.
I know I can edit the tdm.cfg file but where can I find the  exact names of the columns that I can add ?
I tried some but until now I get an empty column ...

**Comment by jziolkowski** [comment, marked as answer]
Since introduction of DeviceName, TDM tries to first retrieve DeviceName from tasmota and then FriendlyName1 if that's not available.

If you want to edit the config file by hand, use "FriendlyName1" as that's the correct property name in tasmota

**Reply by Eddie-BS** [reply]
tnx, that works ;-)

**Reply by mlavelle42** [reply]
Where is the config file for Windows?

**Reply by barbudor** [reply]
Should be in C:\users\yourname\TDM
